### PR TITLE
liblo 0.31

### PR DESCRIPTION
**Pull request by elijahr**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/liblo.rb [hunk 1]
```diff
@@ -36,4 +35,21 @@ def install
...
+      int main() {
+        char version[6];
+        lo_version(version, 6, 0, 0, 0, 0, 0, 0, 0);
+        printf(\"%s\", version);
+        return 0;
+      }",
+    )
+    system ENV.cc, "lo_version.c", "-I#{include}", "-L#{lib}", "-llo", "-o", "lo_version"
+    lo_version = `./lo_version`
+    assert_equal lo_version, "0.31"
```
Comment (SMillerDev) [inline Formula/liblo.rb]: Can you use version.to_str here instead of a hard-coded 0.31

File: Formula/liblo.rb [hunk 2]
```diff
@@ -36,4 +35,21 @@ def install
 
     system "make", "install"
   end
+
+  test do
+    File.write(
```
Comment (SMillerDev) [inline Formula/liblo.rb]: Please look how other formula do this. Generally we use an EOS block

File: Formula/liblo.rb (hunk 1 above)
Comment (elijahr) [reply Formula/liblo.rb]: Updated in 40b0806946735ca15adebaa02d87190c70353a2d

File: Formula/liblo.rb (hunk 2 above)
Comment (elijahr) [reply Formula/liblo.rb]: Thank you! Updated in 40b0806946735ca15adebaa02d87190c70353a2d

**Comment by Bo98** [comment]
@BrewTestBot test this please

**Pull request closed without merging**